Makefile

```diff
@@ -114,521 +105,312 @@ endef
+PHONY_TARGETS += format-src format-src.run $(addsuffix .format-src,$(FORMAT_SRC_FILES)) format-tools format-symbols format-license
+PHONY_TARGETS += $(addsuffix .force-symbols,$(notdir $(wildcard $(BUILD_DIR:$(VERSION)=us)/*.elf))) force-extract context $(addprefix format-symbols.,us hd pspeu saturn $(FORMAT_SYMBOLS_FILES))
+PHONY_TARGETS += git-submodules update-dependencies update-dependencies-all $(addprefix dependencies.,us pspeu hd saturn) python-dependencies graphviz $(DOSEMU_APP)
 PHONY_TARGETS += help get-debug get-phony get-silent
-MUFFLED_TARGETS += $(PHONY_TARGETS) $(MASPSX_APP) $(MWCCGAP_APP) $(MWCCPSP) $(SATURN_SPLITTER_DIR) $(SATURN_SPLITTER_APP) $(EXTRACTED_DISK_DIR)
-MUFFLED_TARGETS += $(DOSEMU_APP) $(ASMDIFFER) $(dir $(M2C_APP)) $(M2C_APP) $(PERMUTER_APP) $(SOTNDISK) $(SOTNASSETS) $(VENV_DIR) $(VENV_DIR)/bin
-.PHONY: $(PHONY_TARGETS)
-# Specifying .SILENT in this manner allows us to set the DEBUG environment variable and display everything for debugging
-#$(DEBUG).SILENT: $(MUFFLED_TARGETS)
-# These are walls of text, so they're redirected to files instead of stdout for debugging
-get-debug: get-phony get-silent
-get-phony:
-	echo ".PHONY:" > make.phony.targets
-	$(foreach target,$(PHONY_TARGETS),echo $(target) >> make.phony.targets;)
-get-silent:
-	echo ".SILENT:" > make.silent.targets
-	$(foreach target,$(MUFFLED_TARGETS),echo $(target) >> make.silent.targets;)
+MUFFLED_TARGETS += $(PHONY_TARGETS) $(MASPSX_APP) $(MWCCGAP_APP) $(MWCCPSP) $(SATURN_SPLITTER_DIR) $(SATURN_SPLITTER_APP) $(EXTRACTED_DISK_DIR) $(ASMDIFFER_APP) $(PERMUTER_APP) $(dir $(M2C_APP)) $(M2C_APP)
+MUFFLED_TARGETS += $(DOSEMU_DIR) tools/dosemu.make.chkpt tools/python-dependencies.make.chkpt tools/graphviz.make.chkpt $(SOTNDISK) $(SOTNASSETS) $(VENV_DIR) $(VENV_DIR)
+.PHONY: $(PHONY_TARGETS) expected clean force-symbols disk disk-prepare disk-debug
+# Specifying .SILENT in this manner allows us to set the VERBOSE environment variable and display everything for debugging
+$(VERBOSE).SILENT: $(MUFFLED_TARGETS)
+
+# These targets have been renamed, but need to remain usable until CI is updated with the new name.
+force_extract: force-extract
+	$(info make: warning: force_extract has been renamed to force-extract and will be removed in the future.)
+force_symbols: force-symbols
+	$(info make: warning: force_symbols has been renamed to force-symbols and will be removed in the future.)
+extract_disk: extract-disk
+	$(info make: warning: extract_disk has been renamed to extract-disk and will be removed in the future.)
+disk_prepare: disk-prepare
+	$(info make: warning: disk_prepare has been renamed to disk-prepare and will be removed in the future.)
+disk_debug: disk-debug
+	$(info make: warning: disk_debug has been renamed to disk-debug and will be removed in the future.)
+dump_disk: dump-disk
+	$(info make: warning: dump_disk has been renamed to dump-disk and will be removed in the future.)
```

Comment:
I renamed these targets so that all target naming was consistent with `-` as a word break.